docs/man/man1/tkn-bundle-pull.1

```diff
@@ -0,0 +1,86 @@
+.TH "TKN\-BUNDLE\-PULL" "1" "" "Auto generated by spf13/cobra" "" 
```

Comment:
I think this is an artefact from before the rename you may want to `git rm -f` it 🙇🏻

Reply:
Sorry, these are hard to squash 😄